en/docs/admin-portal/applications/types/oidc.md

```diff
@@ -0,0 +1,7 @@
+# OIDC Web Application
```

Comment:
Can we make the heading 'Add OIDC Web Application' or 'Register OIDC Web Application'